redis/redis.go

```diff
@@ -187,3 +187,27 @@ type SlowLog struct {
 	// ClientName is the name set via the CLIENT SETNAME command (4.0 only).
 	ClientName string
 }
+
+// Latency represents a redis LATENCY LATEST
+type Latency struct {
+	// Latency event name
+	Event string
```

Comment:
Would this make more sense to call it Name?
```suggestion
	// Name is the name of event.
	Name string
```

Reply:
May-be even `EventType`?